coldfront/plugins/qumulo/forms/UpdateAllocationForm.py

```diff
@@ -18,3 +20,15 @@ def __init__(self, *args, **kwargs):
             required=False,
         )
         self.fields["prepaid_expiration"].disabled = True
+        self.fields["rw_users"].required = self._rw_user_required()
+
+    def _rw_user_required(self) -> bool:
+        required = True
+        if self.allocation_id is not None:
+            allocation = Allocation.objects.get(pk=self.allocation_id)
+            ready_for_deletion_id = AllocationStatusChoice.objects.get(
+                name="Ready for deletion"
+            ).id
+            if allocation.status_id == ready_for_deletion_id:
```

Comment:
This works and will avoid the extra query, but I think we can reduce it even further.  See other comments
```suggestion
            if allocation.status.anme == "Ready for deletion"
```